.pyup.yml

```diff
@@ -2,6 +2,8 @@ schedule: "every day"
 search: False
 update: insecure
 requirements:
-  - tests/e2e/requirements.txt:
+  - tests/e2e:
+      - Pipfile
+      - pipenv.txt
```

Comment:
I don’t think this will work. I think the full path to each file needs to be specified, looking at the docs.

Reply:
Hopefully fixed - went off an untested example from an email exchange with the Pyup folks